fast_barcode_scanner/android/src/main/kotlin/com/jhoogstraat/fast_barcode_scanner/FastBarcodeScannerPlugin.kt

```diff
@@ -150,11 +151,21 @@ class FastBarcodeScannerPlugin : FlutterPlugin, MethodCallHandler, StreamHandler
         }
     }
 
+    /**
+     * each barcode comes with an optional set of 4 points for each of the corners of the scanned code.
+     * In order to send this over the event channel we will serialize this point list as a list of arrays
+     * [[x,y], [x,y], [x,y], [x,y]]
+     */
+    private fun buildPointList(points: Array<Point>?): List<List<Int>>? {
+        return points?.map { point: Point -> listOf(point.x, point.y) }
```

Comment:
This can be shortened to:
```suggestion
return points?.map { listOf(it.x, it.y) }
```

Reply:
👍  will do.